packages/flutter/test/material/expansion_tile_test.dart

```diff
@@ -256,4 +256,130 @@ void main() {
     expect(listTileRect.top, tallerWidget.top - remainingHeight / 2 - 12);
     expect(listTileRect.bottom, tallerWidget.bottom + remainingHeight / 2 + 10);
   });
+
+  testWidgets('ExpansionTile expandedAlignment test', (WidgetTester tester) async {
+    await tester.pumpWidget(MaterialApp(
+      home: Material(
+        child: Center(
+          child: ExpansionTile(
+            title: const Text('title'),
+            expandedAlignment: Alignment.centerLeft,
+            children: <Widget>[
+              Container(height: 100, width: 100),
+              Container(height: 100, width: 80),
+            ],
+          ),
+        ),
+      ),
+    ));
+
+    await tester.tap(find.text('title'));
+    await tester.pumpAndSettle();
+
+    final Rect columnRect = tester.getRect(find.byType(Column).last);
+
+    // The expandedAlignment is used to define the alignment of the Column widget in
+    // expanded tile, not the alignment of the children inside the Column.
+    expect(columnRect.left, 0.0);
+    // The width of the Column is the width of the largest child. The largest width
+    // being 100.0, the offset of the right edge of Column from X-axis should be 100.0.
+    expect(columnRect.right, 100.0);
+  });
+
+  testWidgets('ExpansionTile expandedCrossAxisAlignment test', (WidgetTester tester) async {
+    const Key child0Key = Key('child0');
+    const Key child1Key = Key('child1');
+    // Set the column's alignment to Alignment.centerRight to test CrossAxisAlignment
+    // of children widgets. This helps distinguish the effect of expandedAlignment
+    // and expandedCrossAxisAlignment later in the test.
+    await tester.pumpWidget(MaterialApp(
+      home: Material(
+        child: Center(
+          child: ExpansionTile(
+            title: const Text('title'),
+            expandedAlignment: Alignment.centerRight,
```

Comment:
nit: you could move the comment down here instead, so it's right above the actual parameter :)
```suggestion
            // Set the column's alignment to Alignment.centerRight to test CrossAxisAlignment
            // of children widgets. This helps distinguish the effect of expandedAlignment
            // and expandedCrossAxisAlignment later in the test.
            expandedAlignment: Alignment.centerRight,
```